k8s/totem-express-configmap.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: totem-express-user-service-configmap
+data:
+  SPRING_DATASOURCE_URL: "jdbc:mysql://totem-express-db.ct4egkeoywlc.us-east-1.rds.amazonaws.com/totemexpress?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&zeroDateTimeBehavior=convertToNull&useSSL=false"
```

```suggestion
metadata:
  name: totem-express-order-product-service-configmap
data:
  SPRING_DATASOURCE_URL: "jdbc:mysql://totem-express-produtos-pedidos.ct4egkeoywlc.us-east-1.rds.amazonaws.com/totemexpress?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&zeroDateTimeBehavior=convertToNull&useSSL=false"
```

Mudei o nome do banco pq é o que to usando no terraform 😬